clash-ghc/src-ghc/Clash/GHC/Evaluator/Primitive.hs

```diff
@@ -3230,12 +3230,6 @@ ghcPrimStep tcm isSubj pInfo tys args mach = case primName pInfo of
                     _ -> Nothing
                  _ -> Nothing
 
-  "Clash.Transformations.eqInt"
-    | [ DC _ [Left (Literal (IntLiteral i))]
-      , DC _ [Left (Literal (IntLiteral j))]
-      ] <- args
-    -> reduce (boolToBoolLiteral tcm ty (i == j))
```

Comment:
Does that mean we need `GHC.Classes.eqInt` here?

Reply:
It has a rule already: https://github.com/clash-lang/clash-compiler/blob/1f0db2b41935a656c59c5c8b85b3dd3b603153bc/clash-ghc/src-ghc/Clash/GHC/Evaluator/Primitive.hs#L918